app/models/concerns/reviewable_action_builder.rb

```diff
@@ -153,7 +153,12 @@ def build_action(
     require_reject_reason: false
   )
     actions.add(id, bundle: bundle) do |action|
-      prefix = "reviewables.actions.#{id}"
+      if self.type_source == "core"
+        prefix = "reviewables.actions.#{id}"
+      else
+        prefix = "#{self.type_source.underscore}.reviewables.actions.#{id}"
+      end
+
```

Comment:
Ran into a bug with the `add_action()` helper, this would only need to be fixed once.